### PR TITLE
core_unix.v0.15.1

### PR DESCRIPTION
**Pull request by tov**
Signed-off-by: Jesse Tov <jtov@janestreet.com>

### CODE REVIEW

File: packages/core_unix/core_unix.v0.15.1/opam [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+bug-reports: "https://github.com/janestreet/core_unix/issues"
+dev-repo: "git+https://github.com/janestreet/core_unix.git"
+doc: "https://ocaml.janestreet.com/ocaml-core/latest/doc/core_unix/index.html"
+license: "MIT"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+]
+depends: [
+  "ocaml"                    {>= "4.08.0"}
+  "core"                     {>= "v0.15.1" & < "v0.16"}
```
Comment (mseri) [inline packages/core_unix/core_unix.v0.15.1/opam:14]: Is this correct? It seems that we don't have a `core.v0.15.1`

Comment (mseri) [reply packages/core_unix/core_unix.v0.15.1/opam:14]: Oh, sorry, I did not see https://github.com/ocaml/opam-repository/pull/22261

**Comment by kit-ty-kate** [comment]
Thanks!

**Comment by smuenzel** [comment]
I think the checksum is incorrect on this file, or it has changed since merging to opam.
```bash
wget -O - "https://github.com/janestreet/core_unix/archive/refs/tags/v0.15.1.tar.gz" | sha256sum
6c5fafcd939c1e890295ee86c61e034bdb05f276b89ca2943ffcce0bf10406d2 -
```

**Comment by kit-ty-kate** [comment]
it looks like this has been fixed

**Comment by tov** [comment]
Sorry, that was my mistake when dealing with the erroneous release.

**Pull request merged**